npm/src/db/sql/mssql.ts

```diff
@@ -0,0 +1,57 @@
+type Options = {
```

Comment:
Should be moved inside `mssql` folder into `mssql/index.ts` as its more like a util function exclusive to mssql.